corehq/ex-submodules/couchexport/tests/test_schema.py

```diff
@@ -110,7 +110,7 @@ def test_split_column_ignore_extras(self):
 
     def test_split_column_missing(self):
         self.post_it()
-        self._test_split_column([[None, None, None, None, None]])
+        self._test_split_column([[None, None, None, None, SCALAR_NEVER_WAS]])
```

Comment:
The value gets passed through to `column.get_data`. For SplitColumns it expands the value into one column for each option (plus an optional extra column for any unrecognised options). See https://github.com/dimagi/commcare-hq/pull/10985/files/9819a8cca49d5cb500e851b8dcd828853c89f8da..49181ca6abd80f1813f5cb1e1e161108c189645c#r57936473 for where this happens